net/https-dns-proxy/files/etc/init.d/https-dns-proxy

```diff
@@ -131,13 +130,11 @@ append_bootstrap() {
 	[ "$ipv6_resolvers_only" -eq 0 ] && xappend '-4'
 }
 
-resolver_health_check() { resolveip -t 3 one.one.one.one >/dev/null 2>&1; }
-
 boot() {
 	ubus -t 30 wait_for network.interface 2>/dev/null
 	on_boot_trigger=1
-	rc_procd start_service 'on_boot' && rc_procd service_started 'on_boot'
-	resolver_health_check || rc_procd stop_service 'on_boot'
+	rc_procd start_service 'on_boot' && service_started 'on_boot'
+	is_resolver_working || rc_procd stop_service 'on_boot' && service_stopped 'on_boot'
```

Comment:
Hello,
`service_stopped 'on_boot'` will likely get executed every time.

```
BusyBox v1.30.1 (Ubuntu 1:1.30.1-4ubuntu6.4) built-in shell (ash)
Enter 'help' for a list of built-in commands.

~ $ true || echo foo && echo bar
bar
~ $ false || echo foo && echo bar
foo
bar
~ $ 
```
Is this intentional?

Reply:
The above is from busybox on Ubuntu 20.04, I don't have the router handy.

Reply:
Thank you for spotting it, not intentional, I'll push a fix!